$NetBSD: patch-ak,v 1.1 2006/03/16 09:08:55 ghen Exp $

Fix build error on Solaris with SunStudio compilers (symbol referencing error),
provided by PR pkg/33077.  Also submitted upstream as bacula bug #567.

--- src/dird/ua_dotcmds.c.orig	2006-03-16 09:53:18.000000000 +0100
+++ src/dird/ua_dotcmds.c
@@ -37,7 +37,7 @@ extern const char *client_backups;
 extern int console_msg_pending;
 
 /* Imported functions */
-extern int do_messages(UAContext *ua, const char *cmd);
+extern void do_messages(UAContext *ua, const char *cmd);
 extern int quit_cmd(UAContext *ua, const char *cmd);
 extern int qhelp_cmd(UAContext *ua, const char *cmd);
 extern int qstatus_cmd(UAContext *ua, const char *cmd);
